Add footnote to Platform Support for issue 1361 (#201)
NVIDIA/cloud-native-docs: gpu-operator/life-cycle-policy.rst

   * - NVIDIA GDRCopy Driver
     - `v2.5.0 <https://github.com/NVIDIA/gdrcopy/releases>`__

.. _known-issue:

   :sup:`1`

>    Known Issue: For drivers 570.124.06, 570.133.20, and 570.148.08, 

Reviewer: @tariq1890, will all these driver versions work with 25.3.0? 

I'm just thinking that we don't list them as a supported driver, will it cause more confusion to have the complete list of impacted drivers, or to just say the ones we list as supported are impacted?

Reviewer: we also dont list 570.86.15 as supported, should we?

...sorry if i'm over thinking this :)

Reviewer 2: That's a very valid question, thanks for highlighting this! I think it would make sense to add those TRD versions back to the Platform support and then mention `570.86.15` as the driver version unaffected by this bug.

Author: I think it's ok. We should update the docs once the other driver has been released.

   GPU workloads cannot be scheduled on nodes that have a mix of MIG slices and full GPUs. 
   This manifests as GPU pods getting stuck indefinitely in the ``Pending`` state. 

>    The solution is to downgrade the driver to version 570.86.15 to work around this issue.

Reviewer: ```suggestion
   It's recommended that you downgrade the driver to version 570.86.15 to work around this issue.
```

just a nit suggestions. I think the "it's recommended" wording used in other places in the site, so i thought it might help add some consistency
